app/messaging/save-to-database.js

```diff
@@ -1,17 +1,27 @@
 const db = require('../data')
 
+const formatDateToUK = (date) => {
+  const day = String(date.getDate()).padStart(2, '0')
+  const month = String(date.getMonth() + 1).padStart(2, '0')
+  const year = date.getFullYear()
+  return `${day}-${month}-${year}`
+}
+
 const saveToDatabase = async (message) => {
   try {
+    const formattedDate = formatDateToUK(new Date(message.body.requestedDate))
```

Comment:
I would pass the date as a string  not a date object as you have done here and then let function be responsible for pulling out the day month year and then return as a new date object